Product/SoapUI_Test/RegressionSuite/Standard/EntityDocQuerySecuredInterfaceTest/EntityDocQuerySecuredInterfaceTest-soapui-project.xml

```diff
@@ -392,27 +387,27 @@ if(context.findProperty("PassThruOn").toBoolean()){
       <con:properties>
         <con:property>
           <con:name>startDate</con:name>
-          <con:value>2013-11-13T16:53:54Z</con:value>
+          <con:value>2016-05-12T17:36:31Z</con:value>
         </con:property>
         <con:property>
           <con:name>endDate</con:name>
-          <con:value>2013-11-13T17:03:54Z</con:value>
+          <con:value>2016-05-12T17:46:31Z</con:value>
         </con:property>
         <con:property>
           <con:name>sigDate</con:name>
-          <con:value>11/13/2013 16:53:54</con:value>
+          <con:value>05/12/2016 17:36:31</con:value>
         </con:property>
         <con:property>
           <con:name>expireDate</con:name>
-          <con:value>2013-12-13T00:00:00Z</con:value>
+          <con:value>2016-06-11T00:00:00Z</con:value>
         </con:property>
         <con:property>
           <con:name>FullPatientID</con:name>
           <con:value>'D123401^^^&amp;amp;1.1&amp;amp;ISO'</con:value>
         </con:property>
         <con:property>
           <con:name>PassThruOn</con:name>
-          <con:value>true</con:value>
+          <con:value>false</con:value>
```

Comment:
This should be true for Passthru only. Since it is a Standard project, I am removing  Passthru to satisfy sailaja's comments.